### ISSUE TITLE
Codification behaviour mystery (bug?)

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap--Build-Your-Own-Blocks#1874]
The following ignores the addition:

![image](https://user-images.githubusercontent.com/2138696/31044798-b2b8a28c-a5d6-11e7-892e-ed4285e3cd11.png)

And yet a more elaborate version of this after loading the codification example project works fine.

Tested in release and dev version. Here's the project:

https://snap.berkeley.edu/snapsource/snap.html#present:Username=toontalk&ProjectName=codification%20problem

**Comment by towerofnix** [comment]
I think that the "+" block needs to be ringified? Otherwise you're just passing 0 (() + () = 0) to the MAP TO block.

**Comment by jmoenig** [comment]
thanks, @towerofnix, for the excellent analysis!

**Comment by ToonTalk** [comment]
Thanks that fixed it. But it is an easy mistake to make since commands are automatically ringified while operators are not.

**Comment by towerofnix** [comment]
Yeah, I'm pretty sure the idea behind stack-rings being replaced (rather than filled) when a reporter is dropped on top is that Snap<i>!</i> thinks you mean that the reporter you dropped is meant to return the stack block that will be passed to that block (e.g., "set (my script) to { move (10) steps }; run (my script)"). But, of course, that's not always what you *actually* mean, as seen here. (I've also run into this reporter-codification issue myself.)
